lib/snake/controller.rb

```diff
@@ -11,26 +11,48 @@ def initialize
     @play = false
   end
 
-  # rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComplexity
   def check_input
     Window.on :key_down do |event|
       case event.key
       when 'left'
-        @direction = 'left'  if @last != 'right'
+        turn_left
       when 'right'
-        @direction = 'right' if @last != 'left'
+        turn_right
       when 'up'
-        @direction = 'up'    if @last != 'down'
+        turn_up
       when 'down'
-        @direction = 'down'  if @last != 'up'
+        turn_down
       when 'escape'
-        exit
+        escape_game
       when 'p'
-        @play = true
+        start_game
       end
     end
   end
-  # rubocop:enable Metrics/MethodLength, Metrics/CyclomaticComplexity
+
+  def turn_left
+    @direction = 'left'  if @last != 'right'
+  end
+
+  def turn_right
+    @direction = 'right' if @last != 'left'
+  end
+
+  def turn_up
+    @direction = 'up'    if @last != 'down'
+  end
+
+  def turn_down
+    @direction = 'down'  if @last != 'up'
+  end
```

Comment:
Not sure if you want to keep the indents to the `if`s here, what does RuboCop say about it?

Reply:
It actually had no gripes with this, but I personally want to go back to normal indentations now seeing as they are more isolated now.